b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/localResErrors.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+function error404(res, msg) {
+    return res.status(404).json(msg);
+}
+
+function notPostError(res) {
+    return error404(res, 'Cannot request data directyly.');
```

Comment:
check the spellings here ?  `directly` ?

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/00fcf2af0c800b9bfc234e95534cb746916ccd07